app/Helpers/Helpers.php

```diff
@@ -626,6 +626,9 @@ function formatFilesetMeta($fileset)
                     $fileset[$metadata['name']] = $metadata['description'];
                 }
             }
+            // December 2022, this is temporary to support older versions of 5fish applications.
+            // Revisit in one year after discussing with James Thompson.
+            $fileset['stock_no'] = null;
```

Comment:
I have added this fixed attribute because the method: formatFilesetMeta creates a fileset attribute dynamically for each `bible_fileset_tags` record. 

@bradflood , maybe, we could create a new ticket to move the method: `formatFilesetMeta` to BibleFileset Model class because I think there is a best place to locate `formatFilesetMeta` method.

Reply:
yes, please create the ticket, and work on it when/if you have time

Reply:
@bradflood , ok let me create the ticket.